eng/PublishSymbols.proj

```diff
@@ -0,0 +1,59 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="RunPublishSymbols">
+  <PropertyGroup>
+    <!-- TFM doesn't matter. These settings are required to make NuGet happy so we can restore required MSBuild packages. -->
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <DisableImplicitFrameworkReferences>true</DisableImplicitFrameworkReferences>
+    <ManifestsPath>artifacts\manifests\</ManifestsPath>
+    <DisablePackageReferenceRestrictions>true</DisablePackageReferenceRestrictions>
+  </PropertyGroup>
```

Comment:
Could add a `$(SymbolExpirationInDays)` default if we want. Far as I can see, no value is passed to the symbol servers unless there's a `SymbolExpirationInDays` pipeline or queue time variable set i.e. something in the environment when this project is built. Thoughts?

Reply:
Let's match what arcade does, so set to 3650. https://github.com/dotnet/arcade/blob/fc9dd270f84398306ec28d723f0c4d032e3f2803/eng/common/PublishToSymbolServers.proj#L38